src/modules/target.js

```diff
@@ -101,7 +105,11 @@ export default {
       // Add delay to prevent Firefox from flickering
       setTimeout(
         function updateSrc () {
+          if (this.srcset) {
+            this.el.setAttribute('srcset', '')
```

Comment:
Maybe [Element.removeAttribute](https://developer.mozilla.org/en-US/docs/Web/API/Element/removeAttribute) is more appropriate here?